packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -27,9 +27,9 @@ spec:
       profile: windows
       nodeAgentFluentbit:
         filterKubernetes:
-          Kube_Tag_Prefix: "kuberentes.C.var.lib.rancher.rke.log."
+          Kube_Tag_Prefix: "{{ template "windowsKubernetesFilter" . }}.var.lib.rancher.rke.log."
```

Comment:
**Note to reviewers:** capital `C` is required, which is why we convert `c:` to `C:` now.